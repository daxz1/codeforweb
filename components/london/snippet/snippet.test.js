/* global describe, it, expect */
import React from 'react';
import renderer from 'react-test-renderer';
import Snippet from './snippet';

describe('[RENDER] components/london/snippet/snippet.js', () => {
  it('Header Component renders', () => {
    const component = renderer.create(
      <Snippet
        img=""
        title="A year after the mutiny, HSBC plots its next investment banking overhaul"
        snippet="Former JPMorgan executive Greg Guyett is the latest to try his hand at helping HSBC win a bigger share of global investment banking fees"
        tags={['Interactive', 'Asset Management', 'Brexit']}
      />,
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
