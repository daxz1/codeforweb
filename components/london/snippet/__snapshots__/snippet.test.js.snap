// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[RENDER] components/london/snippet/snippet.js Header Component renders 1`] = `
<article>
  
  <h3>
    A year after the mutiny, HSBC plots its next investment banking overhaul
  </h3>
  <p>
    Former JPMorgan executive Greg Guyett is the latest to try his hand at helping HSBC win a bigger share of global investment banking fees
  </p>
  <div
    className="tags"
  >
    <ul>
      <li>
        Interactive
      </li>
      <li>
        Asset Management
      </li>
      <li>
        Brexit
      </li>
    </ul>
  </div>
</article>
`;
